wgsl/index.bs

```diff
@@ -3393,6 +3393,17 @@ short_circuit_or_expression
 
 # Statements TODO # {#statements}
 
+## Block Statement ## {#block-statement}
+
+A block statement is a brace-enclosed group of zero or more statements.
+A declaration introduced in a block statement is only [=in scope=] until the
```

Comment:
By current wording of a declaration, scope is a property of the _identifier_ for a declaration.

Can we tighten this as follows:


> When a declaration is one of those statements, its identifier is [=in scope=] from the start of the next statement until the end of the compound statement.

or:

> When a declaration is one of those statements, the  [=scope=] of the declaration's identifier extends from the statement after the declaration until the end of the compound statement.

I think I prefer the first.